server/core_account.go

```diff
@@ -72,7 +73,7 @@ func GetAccount(ctx context.Context, logger *zap.Logger, db *sql.DB, statusRegis
 	var updateTime pgtype.Timestamptz
 	var verifyTime pgtype.Timestamptz
 	var disableTime pgtype.Timestamptz
-	var deviceIDs pgtype.VarcharArray
+	var deviceIDs pgtype.Array[string]
```

Comment:
there is `FlatArray[T]` which is just a newtype for `[]T` and doesn't support multidimentional arrays because of that, but otherwise should work just as well here